format/avro/avro_ocf.md

```diff
@@ -0,0 +1,7 @@
+Supports reading Avro Object Container Format (OCF) files based on the [1.11.0 specification](https://avro.apache.org/docs/current/spec.html#Object+Container+Files).
+
+Capable of handling null, deflate, and snappy codecs for data compression.
+
+Limitations:
+ - Schema does not support self-referential types, only built-in types.
+ - Decimal logical types are not supported for decoding, will just be treated as their primitive type
```

Comment:
👍  this puts the other format documentation to shame. Good idea to have links specification etc, if your using fq for something if quite likely you want to have has much details and help as possible :)

I've usually collect links and useful tools etc as comments in the go-files but much of that can probably move out to documentation.